app/models/group.rb

```diff
@@ -1,7 +1,8 @@
 # Maintains group information for a given user on a specific assignment
 class Group < ApplicationRecord
 
-  after_create :set_repo_name, :build_repository
+  after_create :set_repo_name, :repo_cannot_exist_already
```

Comment:
This actually has to be an `after_create` because the `repo_path` must already be set which is set in `set_repo_name` which must be called after the object is created because it relies on the object's id when autogenerating a repo name. 

I'll update the documentation for this function to clarify the steps involved in repo creation